precommit/core.d/patchfiles.sh

```diff
@@ -327,9 +327,10 @@ function patchcmd_apply
 function patchfile_apply_driver
 {
   declare patchfile=$1
+  declare gpg=$2
 
   if declare -f ${PATCH_METHOD}_apply >/dev/null; then
-    "${PATCH_METHOD}_apply" "${patchfile}"
+    "${PATCH_METHOD}_apply" "${patchfile}" "${gpg}"
```

Comment:
this could run when PATCH_METHOD is `gitapply`, but it looks like `gitapply_apply` hasn't been updated for the additional parameter. same with `patchcmd` and `patchcmd_apply` respectively.